taipy/gui/viselements.json

```diff
@@ -846,13 +846,14 @@
                     {
                         "name": "editable[<i>column_name</i>]",
                         "type": "bool",
-                        "default_value": "editable",
-                        "doc": "Indicates, if False, that the indicated column cannot be edited when editable is True."
+                        "default_value": "<i>editable</i>",
```

Comment:
or is the i tag the way to remove it automagically ?

Reply:
No. It's the convention for referring to a property name.